src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -1391,6 +1391,10 @@
   - name: Add a customScript extension to VM(s) specified by --ids.
     text: |
         az vm extension set -n customScript --publisher Microsoft.Azure.Extensions --ids {vm_id}
+  - name: Add an extension and enable automatic upgrade.
```

Comment:
pls use more accurate description from Swagger to make the description clear.  From Swagger
```
the extension should be automatically upgraded by the platform if there is a newer version of the extension available.
```